### PR TITLE
Product details text #93

### PR DESCRIPTION
**Pull request by 3rdvision**
related to: #93

based on margarida's modals ui

### CODE REVIEW

File: bundles/config.en_us.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "ripe_commons.holder.holder-second-label.label": "3D",
 
     "ripe_commons.modal.cancel": "Cancel",
+    "ripe_commons.modal.got_it": "Got it",
```
Comment (gcandal) [inline bundles/config.en_us.json]: Why this change?

Comment (3rdvision) [reply bundles/config.en_us.json]: It was using it for the label of the button in ripe-white but now it's hardcoded, I will erase this

File: vue/store.js [hunk 2]
```diff
@@ -36,6 +36,11 @@ export const store = new Vuex.Store({
         currentFrame: null,
         error: null,
         previous: null,
+        detailsTitle: null,
```
Comment (joamag) [inline vue/store.js]: @gcandal We need to move this RIPE White specific store information out of here. Proper inheritance or composition should be used for the specific details of RIPE White. Also, @gcandal update https://github.com/ripe-tech/ripe-white/issues/93 implementation part to talk about the update of the store.

**Comment by 3rdvision** [comment]
closing this branch as store variables will, instead of here, be now defined in the vuex store setter `_getStoreDef` of ripe-white
